src/Infrastructure/Persistence/Initialization/DatabaseInitializer.cs

```diff
@@ -53,7 +53,7 @@ await scope.ServiceProvider.GetRequiredService<ApplicationDbInitializer>()
             .InitializeAsync(cancellationToken);
     }
 
-    private async Task InitializeTenantDbAsync(CancellationToken cancellationToken)
+    private async Task InitializeRootTenantDbAsync(CancellationToken cancellationToken)
```

Comment:
There is no such thing as a "RootTenantDb"... there is only one TenantDb...

Reply:
This initializes the Root Tenant

Reply:
No, it initializes the Tenant Database.